Formula/needle.rb

```diff
@@ -12,11 +12,14 @@ class Needle < Formula
     sha256 "ea9e38d739acc83ca9a58ba10aef72456e73f22e790462f243192db7aa0a814d" => :mojave
   end
 
-  depends_on xcode: ["11.3", :build]
+  depends_on xcode: ["11.6", :build]
   depends_on xcode: "6.0"
 
   def install
-    system "make", "install", "BINARY_FOLDER_PREFIX=#{prefix}"
+    system "make", "archive_generator"
+    system "install_name_tool", "-change", "@executable_path/lib_InternalSwiftSyntaxParser.dylib", "@executable_path/../libexec/lib_InternalSwiftSyntaxParser.dylib", "./Generator/bin/needle"
```

Comment:
This really needs to be configurable upstream, not patched in the homebrew formula

Reply:
Thanks for the review. By `upstream` do you mean to handle things in our repo instead (i.e. Needle's Makefile)?

Reply:
Yes, the install location of the library should be configurable to allow users (like brew) to follow best-practices.